src/api/config.js

```diff
@@ -0,0 +1,17 @@
+export const CLIENT_ID = "23108a35dd584426b852cfc330a4e481"
+export const REDIRECT_URI = "http://localhost:3000/home"
```

Comment:
I think you'll need to change this, so it can can be used in the prod as well. We can use the `document.location` to grab the current path instead of keep it hardcoded here